spec/core/finalization/schemas/FinalizationProofDTO.yml

```diff
@@ -0,0 +1,23 @@
+type: object
+required:
+  - version
+  - finalizationEpoch
+  - finalizationPoint
+  - height
+  - hash
+  - messageGroups
+properties:
+  version:
+    type: integer
+  finalizationEpoch:
+    type: integer
+  finalizationPoint:
+    type: integer
```

Comment:
@fboucquez check if this is fine for java, those are uint32

Reply:
it needs `format: int64` but I would like them to the defined schemas: Example:

```
finalizationEpoch:
    $ref: "../../../schemas/FinalizationEpoch.yml"
  finalizationPoint:
    $ref: "../../../schemas/FinalizationPoint.yml"

```